_data/form.yml

```diff
@@ -0,0 +1,831 @@
+      shows: proposed-number-basements
+
+    - label: Number of parking spaces
+      shows: proposed-parking-spaces
+
+    - label: Number of bicycle spaces
+      shows: proposed-bicycle-spaces
+
+- title: Fill in your proposed changes.
+  fields:
+  - group: proposed-parking-gsf
+    fields:
+    - label: Proposed parking space
+      type: number
+      unit: gross square feet
+
+  - group: proposed-residential-gsf
+    fields:
+    - label: Proposed residential space
+      type: number
+      unit: gross square feet
+
+  - group: proposed-commercial-gsf
+    fields:
+    - label: Proposed retail commercial space
+      type: number
+      unit: gross square feet
+
+  - group: proposed-affordable
+    fields:
+    - label: Proposed number of affordable dwelling units
+      type: number
+
+  - group: proposed-market-rate
+    fields:
+    - label: Proposed number of market-rate dwelling units
+      type: number
+
+  - group: proposed-number-buildings
+    fields:
+    - label: Proposed number of buildings
+      type: number
+
+  - group: proposed-number-stories
+    fields:
+    - label: Proposed number of stories
+      type: number
+
+  - group: proposed-number-basements
+    fields:
+    - label: Proposed number of basements or cellars
+      type: number
+
+  - group: proposed-parking-spaces
+    fields:
+    - label: Proposed parking spaces
+      type: number
+
+  - group: proposed-bicycle-spaces
+    fields:
+    - label: Proposed bicycle spaces
+      type: number
+
+- title: Fire protection
+  fields:
+  - label: Does the property have fire sprinklers?
+    type: radio
+    options:
+    - label: "Yes"
+      shows: sprinklers
+
+    - label: "No"
+      hides: sprinklers
+
+  - group: sprinklers
+    fields:
+    - label: Which areas of the property do the sprinklers cover?
+
+  - label: Does the property have fire alarms?
+    type: radio
+    options:
+    - label: "Yes"
+      shows: alarms
+
+    - label: "No"
+      hides: alarms
+
+  - group: alarms
+    fields:
+    - label: Which areas of the property do the alarms cover?
+
+- title: Worker's compensation insurance
+  fields:
+  - label: What best describes your worker's compensation insurance coverage for this project?
+    type: radio
+    options:
+    - label: "I am self-insured and have a certificate of consent."
+      hides: comp-II
+      shows: comp-I
+
+    - label: "I have third-party insurance."
+      shows: comp-II
+
+    - label: "I plan to hire a contractor who will follow worker's compensation laws."
+      hides: comp-II
+      shows: comp-V
+
+    - label: "I don't need insurance because I'm not hiring anyone."
+      hides: comp-II
+      shows: comp-IV
+
+    - label: "I don't need insurance because the work will cost under $100."
+      hides: comp-II
+      shows: comp-III
+
+  - group: comp-II
+    fields:
+    - label: Name of insurance carrier
+
+    - label: Policy number
+      type: number
+
+- title: Seismic work
+  fields:
```

Comment:
Add a description like: "All houses in San Francisco need to go through seismic retrofitting."